frontend/feed.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Quadcore: Technology News Aggregator</title>
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <!--Import materialize.css-->
+    <link type="text/css" rel="stylesheet" href="dist/css/materialize.min.css" media="screen,projection" />
+    <!--Import custom css file-->
+    <link type="text/css" rel="stylesheet" href="dist/css/feed.css" />
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
+    <!--Nav bar structure-->
+    <div class="navbar-fixed nav-wrapper">
+        <nav class="white">
+            <div class="nav-wrapper container">
+                <a href="#" class="qc-brand-logo">
+                    <img class="qc-logo-icon" src="dist/img/quadcore-logo-text.png">
+                </a>
+                <ul class="qc-nav-right right">
+                    <li>
+                        <a class="dropdown-button valign-wrapper" href="#!" data-beloworigin="true" data-activates="dropdown-nav">
+                            <i class="qc-member-logo material-icons person">person</i>
+                            <span id="username" class="qc-member-name"> username </span>
+                            <i class="material-icons">arrow_drop_down</i>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </div>
+
+    <!--News feed structure-->
+    <section id="feed">
+        <div class="container">
+            <div class="row">
+                <div class="col s12 feed-wrapper">
+                </div>
+            </div>
+            <div class="preloader-wrapper big active">
+                <div class="spinner-layer spinner-blue-only">
+                    <div class="circle-clipper left">
+                        <div class="circle">
+                        </div>
+                    </div>
+                    <div class="gap-patch">
+                        <div class="circle">
+                        </div>
+                    </div>
+                    <div class="circle-clipper right">
+                        <div class="circle"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Dropdown Stucture Template -->
+    <ul id="dropdown-nav" class="dropdown-content hoverable">
+        <li>
+            <a class="tooltipped flow-text" data-position="left" data-delay="50" data-tooltip="Add social account for more accurate news feed"
+                href="#!" id="edit-profile">
+                <i class="material-icons">person_pin</i>Profile</a>
```

Comment:
Closign tag </a> should be aligned with the opening tag. This saves a lot of time when someone else reads your code.

Reply:
same below